code/modules/reagents/chemistry/holder.dm

```diff
@@ -211,36 +211,44 @@ var/const/INGEST = 2
 			continue
 		if(!M)
 			M = R.holder.my_atom
+
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
 			//Check if this mob's species is set and can process this type of reagent
 			var/can_process = 0
```

Comment:
This all seems horrendously unnecessary now.

It makes sense for `reaction_check`, still, but not this, anymore.